Makefile

```diff
@@ -0,0 +1,38 @@
+CWD := $(shell pwd)
```

Comment:
Are we going to maintain all the spec files as part of tendrl-api for building say rubygem-sinatra ? Where do we keep the source tar balls of these projects?